lighty-modules/lighty-jetty-server/src/main/java/io/lighty/server/AbstractLightyWebServer.java

```diff
@@ -29,37 +28,26 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public final class LightyJettyWebServer implements WebServer {
-    private static final Logger LOG = LoggerFactory.getLogger(LightyJettyWebServer.class);
+public abstract class AbstractLightyWebServer implements WebServer {
+    private static final Logger LOG = LoggerFactory.getLogger(AbstractLightyWebServer.class);
+    protected static final int HTTP_SERVER_IDLE_TIMEOUT = 30000;
```

Comment:
private

Reply:
Cannot be private, needs to be accessed by subclasses.